app/services/index.js

```diff
@@ -0,0 +1,5 @@
+const quotes = require('./quotes');
+
+module.exports = {
+  quotes
+};
```

Comment:
I would think that this file is not necessary, even the service can be exported from the line where you start to declare it. Something like `exports.getQuote`